hylo/build.sh

```diff
@@ -52,4 +52,13 @@ mv .build/release/ce_temp_dir/* .build/release
 # Note: No need to update rpath for `hc` itself, as it is already $ORIGIN by default.
 rmdir .build/release/ce_temp_dir/
 
+# Remove Swifty build artifacts not required to run
```

Comment:
Would it be easier to just copy the binary and its dependencies (like the cobol thing does?) What else do we need? There's nothing really left in the `.build/release` now, right? Will merge and see though!